internal/services/mssql/helper/sql_extended_auditing.go

```diff
@@ -46,16 +49,22 @@ func ExtendedAuditingSchema() *pluginsdk.Schema {
 					Optional: true,
 					Default:  true,
 				},
+				"storage_account_subscription_id": {
+					Type:         pluginsdk.TypeString,
+					Optional:     true,
+					Sensitive:    true,
+					ValidateFunc: validation.IsUUID,
+				},
```

Comment:
perhaps a weird one, but given we use Resource ID's in most of the Provider, should this be a Subscription ID that we parse & format as needed, rather than just a UUID?